src/System.Data.SqlClient/src/Resources/Strings.resx

```diff
@@ -1042,4 +1042,7 @@
   <data name="LocalDBNotSupported" xml:space="preserve">
     <value>LocalDB is not supported on this Platform.</value>
   </data>
+  <data name="PlatformNotSupported_DataSqlClient" xml:space="preserve">
+    <value>System.Data.SqlClient is not supported on this platform.</value>
+  </data>
```

Comment:
Since SQlClient is new in 2.0 I'm not sure what it means to build for those configurations... @ericstj can you help illuminate what's going on here.. what are these here for?
```
   netstandard1.2;
      netstandard1.3;
      netstandard;
```